pandas/core/internals/blocks.py

```diff
@@ -1507,15 +1507,8 @@ def _nanpercentile(values, q, axis, **kw):
                                        len(values)).reshape(len(values),
                                                             len(qs))
             else:
-
-                try:
-                    result = _nanpercentile(values, np.array(qs) * 100,
-                                            axis=axis, **kw)
-                except ValueError:
-
-                    # older numpies don't handle an array for q
```

Comment:
I'm actually just assuming here that the relevant old numpies can by dropped.  @h-vetinari any idea on this?